src/Pages/Cart/Cart.jsx

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import CartCard from "../../Components/CartCard/CartCard";
+import Nav from "../../Components/Nav/Nav";
```

Comment:
Import from index.js of your components folder

Reply:
Thanks, Rohan. I will do it